examples/deck/example.js

```diff
@@ -218,6 +229,7 @@ export default class Example extends Component<
               <option value="draw90DegreePolygon">draw90DegreePolygon</option>
               <option value="drawRectangle">drawRectangle</option>
               <option value="split">split</option>
+              <option value="split90Degree">split90Degree</option>
```

Comment:
No longer needed

Reply:
This option is used to enable "split" tool with 90 degree mode enabled, why are you suggesting to remove it?